.github/workflows/ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
         include:
           - os: ubuntu-latest
             ocaml-version: 4.08.0
+          - os: ubuntu-latest
```

Comment:
Why two times the os: ubuntu-latest  line ?

Reply:
Normally, the syntax is valid and it should do what we want: test the build on Ubuntu using the latest version of OCaml (see https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations).